src/sentry/api/endpoints/project_user_reports.py

```diff
@@ -102,15 +102,9 @@ def post(self, request, project):
             report.event_user_id = euser.id
 
         try:
-            mapping = EventMapping.objects.get(
-                event_id=report.event_id,
-                project_id=project.id,
-            )
-        except EventMapping.DoesNotExist:
-            # XXX(dcramer): the system should fill this in later
+            report.group = Group.objects.from_event_id(project, report.event_id)
```

Comment:
TIL this is nullable. 🤔   Any idea why?

Reply:
I assume historically, the column is nullable. I don't think practically it's a thing to worry about, but tests failed from this.